front/app/components/IdeaCards/IdeasWithFiltersSidebar/FiltersModal.tsx

```diff
@@ -56,8 +56,9 @@ const FiltersModal = ({
           onClick={onClose}
         />
       }
+      contentBgColor="background"
```

Comment:
To avoid issues with the background not growing to the full height, I defined it on the modal instead of more complicated flexbox trickery.